src/lib/coingecko/constants.ts

```diff
@@ -0,0 +1,66 @@
+import type { AssetId, ChainId } from '@shapeshiftoss/caip'
+import {
+  arbitrumChainId,
+  arbitrumNovaChainId,
+  avalancheChainId,
+  baseAssetId,
+  baseChainId,
+  bchAssetId,
+  bchChainId,
+  bscAssetId,
+  bscChainId,
+  btcAssetId,
+  btcChainId,
+  cosmosAssetId,
+  cosmosChainId,
+  dogeAssetId,
+  dogeChainId,
+  ethAssetId,
+  ethChainId,
+  gnosisAssetId,
+  gnosisChainId,
+  ltcAssetId,
+  ltcChainId,
+  optimismChainId,
+  polygonAssetId,
+  polygonChainId,
+  thorchainAssetId,
+  thorchainChainId,
+} from '@shapeshiftoss/caip'
+import invert from 'lodash/invert'
+
+export const CHAIN_ID_TO_COINGECKO_PLATFORM_ID: Partial<Record<ChainId, string | null>> = {
+  [btcChainId]: null,
+  [bchChainId]: null,
+  [dogeChainId]: null,
+  [ltcChainId]: null,
+  [thorchainChainId]: null,
+  [ethChainId]: 'ethereum',
+  [avalancheChainId]: 'avalanche',
+  [optimismChainId]: 'optimistic-ethereum',
+  [bscChainId]: 'binance-smart-chain',
+  [polygonChainId]: 'polygon-pos',
+  [gnosisChainId]: 'xdai',
+  [arbitrumChainId]: 'arbitrum-one',
+  [arbitrumNovaChainId]: 'arbitrum-nova',
+  [baseChainId]: 'base',
+  [cosmosChainId]: 'cosmos',
+}
```

Comment:
Except for the null values:

We already have those in as an enum called `CoingeckoAssetPlatform`

We might wan't to avoid duplicating those strings?

Also it seems that this mapping can be replaced by consuming `chainIdToCoingeckoAssetPlatform` depending on the need for the null values?

Reply:
- Leverage `CoingeckoAssetPlatform`: c9f20b918a
- Leverage `chainIdToCoingeckoAssetPlatform` and add missing mapping in a0c597b867